nautobot/docs/configuration/authentication/sso.md

```diff
@@ -77,6 +77,17 @@ AUTHENTICATION_BACKENDS = [
 !!! warning
 	You should only enable one social authentication authentication backend. It is technically possible to use multiple backends but we cannot officially support more than one at this time.
 
+
+### Custom Authentication Backends
+
+The default external authentication supported is [social-auth-app-django](https://python-social-auth.readthedocs.io/en/latest/configuration/django.html) as stated above. If you have developed your own external authentication backend, you can specify `SOCIAL_AUTH_BACKEND_PREFIX` to enable the SSO redirect when the login button is clicked. Currently, Nautobot will look for `social_core.backends` within `AUTHENTICATION_BACKENDS` to endable the redirect.Let's take a look at an example for a custom authentication backend available at `custom_auth.backends.custom.Oauth2`.
+
+```python
+SOCIAL_AUTH_BACKEND_PREFIX = "custom_auth.backends"
+```
+
+`SOCIAL_AUTH_BACKEND_PREFIX` was set to `custom_auth.backends` as we may have more than one custom backend. This will enable the SSO redirect for users when they click the login button.
```

Comment:
I'm not quite clear what the first sentence here is saying?

Reply:
I will have this updated once my local tests pass and I push up. Hopefully it's more clear, but let me know.